components/sup/src/manager.rs

```diff
@@ -807,7 +807,7 @@ impl Manager {
 
         outputln!("Starting gossip-listener on {}",
                   self.butterfly.gossip_addr());
-        self.butterfly.start_rsw_mlw(&Timing::default())?;
+        self.butterfly.start_rsw_mlw_smw(&Timing::default())?;
```

Comment:
The function we're in (`run_rsw_imlw_mlw_gsw`) needs to have the `_smw` noted in its name and doc comment (and similarly with *its* callers `sub_run_rsr_imlw_mlw_gsw_smw`, `start_rsr_imlw_mlw_gsw` ).

Reply:
Thanks for catching this.  I'll get it updated.